Included GitDuck in Productivity tools (#320)
viatsko/awesome-vscode: README.md

  - [GitLink](#gitlink)
  - [GitLens](#gitlens)
  - [Git Indicators](#git-indicators)

>   - [GitDuck](#gitduck)

Reviewer: ```suggestion
  - [Duckly](#duckly)
```

![git removed indicator screenshot](https://raw.githubusercontent.com/lamartire/vscode-git-indicators/master/preview/removed.png)
![git modified indicator screenshot](https://raw.githubusercontent.com/lamartire/vscode-git-indicators/master/preview/modified.png)

> ## [GitDuck](https://marketplace.visualstudio.com/items?itemName=gitduck.code-streaming)

Reviewer: ```suggestion
## [Duckly](https://marketplace.visualstudio.com/items?itemName=gitduck.code-streaming)
```

> > GitDuck is a video chat tool built for developers. It enables you to talk, share your code and do pair programming with ease.

Reviewer: ```suggestion
> Duckly is a video chat tool with real-time collaboration that allows you to do pair programming with people using different IDEs
```

> ![GitDuck pair programming](https://storage.googleapis.com/gitduck/img/5-img-collaborate-fallback-o.png)

Reviewer: ```suggestion
![Duckly pair programming](https://storage.googleapis.com/gitduck/img/5-img-collaborate-fallback-o.png)
```

> ![GitDuck video chat for developers](https://storage.googleapis.com/gitduck/img/1-img-general-fallback-o.png)

Reviewer: ```suggestion
![Duckly video chat for developers](https://storage.googleapis.com/gitduck/img/1-img-general-fallback-o.png)
```
